SuffolkLITLab/docassemble-MAEvictionMoratorium
# Increase custom expiration date to 31 days. Fixes #48

[Edit: Wondering if this should be 35 days or something to give people a few days after the rent cycle to rethink things last minute.]

Once a docassemble bug is fixed, this can be tested. This test will take a bit more than a week. Fixes #48.

Test:
1. Have one cosigner WITH A VERY UNIQUE NAME so you can keep track of it and remember it a week later.
1. Email or text them the link for the signature.
1. Get all the way to the end.
1. Sign for the cosigner.
1. KEEP THAT EMAIL OR TEXT MESSAGE. MARK IT AS SPECIAL AND KEEP TRACK OF IT.
1. With a good margin of extra time just in case, a little under exactly 7 days (168 hours) use the link and make sure it takes you to the end of the interview again.
1. Wait a week and an hour (or some amount of time over ~7~ 31 days - ~168~ more than 744 hours).
1. Go to that email or message and follow the link again.
1. You should not be able to get to the interview at all.
